cowprotocol/dune-sync
# Introduce new logic for syncing raw batch data

This PR introduces an alternative way of syncing raw batch data on Dune, by relying on the dune api for uploading data as csv files.

The main ideas:
- It creates one table per month
- in order to ensure all data is uploaded, on the first day of each month, it recomputes the previous month's table and re-uploads it
- it finally introduces the auction id and environment as entries in the` batch_data` table!

To test it, one can run

`python -m src.main --sync-table batch_data `

and check the results by running the following query on Dune:

`select * from dune.cowprotocol.dataset_test_batch_rewards_ethereum_2024_11 order by block_deadline DESC`
